pallets/invulnerables/src/benchmarking.rs

```diff
@@ -23,7 +23,7 @@ use super::*;
 #[allow(unused)]
 use crate::Pallet as InvulnerablesPallet;
 use {
-    frame_benchmarking::{account, impl_benchmark_test_suite, v2::*, BenchmarkError},
```

Comment:
Why was this import removed? I do not see any other change in this file.

Reply:
not sure but I guess it was not needed (and the CI was telling me so). Maybe the benchmarking macros changed